docs/api/decorators.md

```diff
@@ -25,11 +25,11 @@ def view_requiring_flag(request):
     return HttpResponse('flag was set')
 
 @flag_check('MY_OTHER_FLAG', False)
-def view_when_flag_is_not_set(request):
+def view_when_other_flag_is_set(request):
     return HttpResponse('flag was set')
 
 def other_view(request):
-    return HttpResponse('flag was not set')
+    return HttpResponse('always available')
```

Comment:
This is not accurate — `other_view`' is only served if the [`flag_check` on line 34 fails](https://github.com/cfpb/django-flags/pull/26/files#diff-47b7fff2c25e1205c4bf93c1a6455438R34). This means that `MY_FLAG_WITH_FALLBACK` did not evaluate to `True`. 

I agree that these examples should be a little bit clearer — I like the idea of `other_view` being fully isolated from the flag checking around `view_with_fallback,` but it would be more accurately titled `fallback_view` or something like, and then return `'MY_FLAG_WITH_FALLBACK was not True'`.